spec/models/presentation_spec.rb

```diff
@@ -1,5 +1,89 @@
+# spec/models/presentation_spec.rb
 require 'rails_helper'
 
 RSpec.describe Presentation, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:person) { FactoryGirl.create(:person) }
+  let(:event)  { FactoryGirl.create(:event) }
+
+  describe 'validate model' do
+    subject { FactoryGirl.create(:presentation) }
+
+    it { is_expected.to validate_presence_of :topic }
+    it { is_expected.to validate_presence_of :duration }
+
+    it { is_expected.to belong_to :person }
+    it { is_expected.to belong_to :event }
+
+    it { is_expected.to respond_to :presenter_name }
```

Comment:
But not `person_or_presenter`? I don't think the `respond_to` adds value here as we'll be testing the methods below anyways. Instead of adding another check for `person_or_presenter` I'd just remove this altogether.